### DISCUSSION TITLE
Error using node.js as webserver with a simple openseadragon HTML file

### DISCUSSION THREAD

**Discussion by ghost** [openseadragon/openseadragon#2103] [Q&A] [answered]
Greetings,
I'm working on a small project (not the project itself is small but the computers it should eventually run on (Raspberry Pi's)) and trying to make a simple HTML file that shows only one (relative) small image with OSD. However, somehow this simple setup won't work and results in an error that can only be seen with chrome browser in F12 modus (Uncaught SyntaxError: Unexpected token '<' openseadragon.js:1)

I stripped the HTML file as much as possible and the error remains until I remove the <script> tag completely, leaving only an almost empty `<head>` and almost empty `<body>`. 

I guess this might not be an openseadragon issue at all but more related to the fact that I use node.js as webserver. But i'm puzzled about my opinion. 

Any ideas / thoughts about this. TIA. 

```
<!doctype html>
<html>
  <head>
    <script src="openseadragon/openseadragon.js"></script>   
  </head>   
  <body>
    BODY
  </body>
</html>
```

![node_osd](https://user-images.githubusercontent.com/39137738/152518603-3b264509-e199-462d-a1cf-b8bd3f58beb6.PNG)

**Comment by iangilman** [comment]
Try loading openseadragon.js directly into the browser. It looks like your server is giving you an html-based error rather than the JavaScript file. That could either be because you have the path wrong, or because your server doesn't know that it should support .js files, or something. 

It's been 12 days, so hopefully you got it figured out, but I figured I'd respond just in case!

**Comment by ghost** [comment, marked as answer]
Thanks for having a look Ian. Indeed this has been solved and indeed my issue had to do with wrong content-type for the javascript files. Since this is my first node.js project I had to learn that part as well,  contentType = 'text/javascript made it work.

**Reply by iangilman** [reply]
Nice! :)
